pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller-test.go

```diff
@@ -135,19 +135,13 @@ var _ = Describe("{{ .Resource.Kind }} controller", func() {
 				}
 				
 				err = k8sClient.Create(ctx, {{ lower .Resource.Kind }})
-				if err != nil {
-					Expect(err).To(Not(HaveOccurred()))
-				}
+				Expect(err).To(Not(HaveOccurred()))
 			} 
 
 			By("Checking if the custom resource was successfully created")
 			Eventually(func() error {
 				found := &{{ .Resource.ImportAlias }}.{{ .Resource.Kind }}{}
-				err = k8sClient.Get(ctx, typeNamespaceName, found)
-				if err != nil {
-					return err
-				}
-				return nil
+				return k8sClient.Get(ctx, typeNamespaceName, found)
```

Comment:
this is the same as 
https://github.com/openshift/operator-framework-olm/blob/3e1dc5ce2547e283949bef2f2cd992cab8ffd730/staging/operator-lifecycle-manager/pkg/controller/operators/openshift/clusteroperator_controller_test.go#L42